### PR TITLE
Add a quote from the president with a picture

### PR DESCRIPTION
**Pull request by annagav**
### What are the relevant tickets?
Fix https://github.com/mitodl/hq/issues/2664

### Description (What does it do?)
Add a quote form the president with a picture.

### Screenshots (if appropriate):
Screenshot 2024-02-05 at 12 00 52 PM

### CODE REVIEW

**Comment by annagav** [comment]
Update:
Screenshot 2024-02-05 at 12 00 52 PM

File: cms/templates/cms/home_page.html [hunk 1]
```diff
@@ -224,6 +224,11 @@ <h3>Join Us at Our Free Online Events</h3>
           <blockquote class="quote-text">“MicroMasters Programs offer unprecedented access points to advanced degrees, as well as to accelerated professional growth in some of the world's most in-demand fields. Through these programs, learners worldwide can gain the experience of a masters-level course of study, but also the confidence of thriving in an MIT-caliber education.”</blockquote>
           <p class="quote-author">Eric Grimson, Vice President of Open Learning, MIT  </p>
         </div>
+        <div class="row quote">
+          <img class="person" src="{% static 'images/kornbluth.png' %}" alt="">
```
Comment (collinpreston) [inline cms/templates/cms/home_page.html:228]: I would add something like "Sally Kornbluth, MIT President" for the alt text.

**Comment by pdpinch** [comment]
Thanks @annagav [the 2nd version](https://github.com/mitodl/micromasters/pull/5366#issuecomment-1927752813) looks good to me. I think this is good to merge once it's been reviewed by an engineer.

**Review by collinpreston — APPROVED** [review]
Looks good.  One accessibility comment.

**Pull request merged**